datafusion/core/tests/sqllogictests/test_files/ddl.slt

```diff
@@ -448,6 +448,16 @@ CREATE TABLE table_without_values(field1 BIGINT NULL, field2 BIGINT NULL);
 statement ok
 CREATE TABLE IF NOT EXISTS table_without_values(field1 BIGINT, field2 BIGINT);
 
+statement ok
+CREATE EXTERNAL TABLE aggregate_simple STORED AS CSV WITH HEADER ROW LOCATION 'tests/data/aggregate_simple.csv'
+
+statement ok
+CREATE EXTERNAL TABLE IF NOT EXISTS aggregate_simple STORED AS CSV WITH HEADER ROW LOCATION 'tests/data/aggregate_simple.csv'
```

Comment:
According to the previous cases I think we need to create this table again without `IF NOT EXIST` and expect an error

Reply:
fixed,thanks again.
`CREATE EXTERNAL TABLE`  
need to check recreate the same EXTERNAL table